### DISCUSSION TITLE
CherryChèvre: A Fine-Grained Dataset for Goat Detection in Natural Environments

### DISCUSSION THREAD

**Discussion by Phirxian** [dataset-ninja/dataset-ninja#6] [New dataset proposal]
https://www.nature.com/articles/s41597-023-02555-8

**Comment by grokhi** [comment]
Thank you for your request. We have added the dataset to our TODO list.

**Comment by grokhi** [comment]
@Phirxian  The dataset has been added
https://datasetninja.com/cherry-chevre
